src/builder/javascript.js

```diff
@@ -1,16 +1,29 @@
 import { transformFile } from 'babel-core'
 import { default as deferred } from 'thenify'
 import { default as compat } from 'babel-plugin-add-module-exports'
-import { default as es2015 } from 'babel-preset-es2015'
-import { default as umd } from 'babel-plugin-transform-es2015-modules-umd'
+import { default as ecmascript } from 'babel-preset-latest'
+
+const babelrc = process.env.NODE_ENV !== 'test'
 
 export default function configure(pkg, opts) {
   return async function process(name, file) {
-    let presets = [es2015]
-      , plugins = [umd]
+    const { es2017
+          , modules
+          , ['add-module-exports']: addModuleExports
+          } = opts.flags
+
+    let presets = [
```

Comment:
this will cause potential issues around use of a custom presets order in `.babelrc` - which can be around precedence of plugins. 

typical usecase is:
```
{
  "presets": ["es2015", "react", "stage-0"]
}
```

`presets` then gets set using the pre-defined ones and loaded after the .babelrc presets, which causes a problem and overrides / removes stuff like `stage-0` that needs to stay last.

we can add `opt.babelrc` as only source of truth and pass an empty presets array when set, leaving control at the project level.